.changeset/popular-scissors-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
+---
+
+Adapt admin-generator to generate files without barrel react imports
```

Comment:
I just realised that this is probably a problem. Previously, we required importing React. Now we ban it.

This means all projects have to switch to the non-barrel version or the linting will fail for the generated files. That's a breaking change. So I fear we must wait for v8 to do this

Reply:
When the generated files would be re-generated once, it would just be fine or? This should not be breaking, i thought?

Reply:
As discussed, we just disable this rule ´"react/react-in-jsx-scope": "off"` to be able to remove the brarrel imports